xarray/core/common.py

```diff
@@ -134,6 +134,12 @@ def __array__(self: Any, dtype: DTypeLike = None) -> np.ndarray:
     def __repr__(self) -> str:
         return formatting.array_repr(self)
 
+    def _repr_html_(self):
+        if OPTIONS["display_style"] == "classic":
+            classic = repr(self).replace("<", "&lt;").replace(">", "&gt;")
+            return "<pre>{repr}</pre>".format(repr=classic)
```

Comment:
Could use an f-string here:
```suggestion
            return f"<pre>{classic}</pre>"
```

Reply:
I didn't realize that xarray was already supporting >=3.6 :+1: